<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>544</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>812</width>
    <height>544</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EasyTest 2</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Общая информация</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="specialist_lbl">
            <property name="text">
             <string>Специалист</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="specialist"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="test_start_date_lbl">
            <property name="text">
             <string>Дата начала аттестации</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateEdit" name="test_start_date"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="test_end_date_lbl">
            <property name="text">
             <string>Дата окончания аттестации</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDateEdit" name="test_end_date"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Испытательная система</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="system_select_lbl">
            <property name="text">
             <string>Выбор испытательной системы</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="system_select">
            <property name="maxVisibleItems">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="system_info"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Приборы и средства измерения</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListWidget" name="tools_available"/>
        </item>
        <item>
         <widget class="QListWidget" name="tools_selected"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Режимы температуры</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QListWidget" name="tmodes_list"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_add_tmode">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Добавить режим температуры</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Режимы относительной влажности</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QListWidget" name="hmodes_list"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_add_hmode">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Добавить режим относительной влажности</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Аттестация</string>
    </property>
    <addaction name="new_test"/>
    <addaction name="open_test"/>
    <addaction name="separator"/>
    <addaction name="save"/>
    <addaction name="save_as"/>
    <addaction name="settings"/>
    <addaction name="separator"/>
    <addaction name="create_report"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about"/>
    <addaction name="separator"/>
    <addaction name="help"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="new_test">
   <property name="text">
    <string>Новая аттестация</string>
   </property>
   <property name="statusTip">
    <string>Создать новый файл аттестации</string>
   </property>
  </action>
  <action name="open_test">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="statusTip">
    <string>Открыть существующий файл аттестации</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="statusTip">
    <string>Сохранить изменения текущего файла</string>
   </property>
  </action>
  <action name="save_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="statusTip">
    <string>Сохранить текущий файл под новым именем</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="about">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="statusTip">
    <string>Информация о программе</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="help">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="statusTip">
    <string>Справочная информация для пользователей</string>
   </property>
  </action>
  <action name="settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
   <property name="statusTip">
    <string>Настройки программы</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="statusTip">
    <string>Выход из программы</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Расчитать результаты</string>
   </property>
  </action>
  <action name="calculate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Рассчитать результаты</string>
   </property>
   <property name="statusTip">
    <string>Произвести расчёт результатов аттестации на основе предоставленной информации</string>
   </property>
  </action>
  <action name="create_report">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сформировать протокол</string>
   </property>
   <property name="statusTip">
    <string>Сформировать протокол (в протокол будут включены те режимы для которых был произведён успешный расчёт)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
